python/ray/internal/storage.py

```diff
@@ -129,6 +144,10 @@ def get(self, path: str) -> bytes:
                 return f.read()
         except FileNotFoundError:
             return None
+        except OSError as e:
+            if _is_os_error_file_not_found(e):
+                return None
```

Comment:
How about this:

```
except Exception as e:
    if _is_file_not_found(e):
        return None
    raise e
```

Reply:
I prefer to be more explicitly about the exception handling. According the current understanding, `OSError` is the extra exception we need to handle, and `_is_os_error_file_not_found` indicates this is a temporary patch. `Exception` lowers the level of exception handling too much.